mammoth/modules/transformer_decoder.py

```diff
@@ -317,9 +303,9 @@ def from_opts(cls, opts, embeddings, is_on_top=False):
             embeddings,
             opts.max_relative_positions,
             opts.aan_useffn,
-            opts.full_context_alignment,
-            opts.alignment_layer,
-            alignment_heads=opts.alignment_heads,
+            False,
```

Comment:
random hardcoded values in the middle of a list of args is not cool...

3 options:
- kwargs would help. 
- add a comment if you're forcing default values, maybe add a log if the default value conflicts with the user reqs? 
- remove the corresponding args in the constructor

Reply:
I actually want to remove this. The old transformer implementation is no longer needed for anything other than the attention bridge.

I think I can safely remove the decoder, as the AB only needs TransformerEncoderLayer and MultiHeadedAttention. I'm not completely sure what depends on what, so I didn't yet remove anything.

I'm not going to port the attention bridge to x-transformers. As far as I'm concerned, it is obsolete.

Reply:
I don't think we have any reasons (outside of the sentimental) to keep the AB in the impl; but that feels like a discussion with Jörg and Raúl.